index.bs

```diff
@@ -1619,7 +1693,114 @@ QUIC streams.
 Stats {#streaming-stats}
 ------------------------------
 
-TODO
+Durings a streaming session, the sender should send stats with the
+streaming-session-sender-stats-event at the interval the receiver
+requested.  The streaming-session-sender-stats-event message contains
+the following properties:
+
+: streaming-session-id
+:: The ID of the streaming session these stats apply to.
+
+: system-time
```

Comment:
How would the receiver correlate these stats with specific audio or video frames?
Maybe the more relevant timing information is a min-max start time for the corresponding frames.

Reply:
On the other hand, there may be intervals where no frames are being sent.  So maybe there needs to be a duration field, covering the wall clock time that elapsed during the stats snapshot.

Reply:
All the stats are meant to be "duration less" by using cumulative stats such that any two points in time can be compared to compute changes over time.  That way if you lose some samples, you can still have meaningful data, and if you have all the samples, you can use what window you want for averages and other calculations.

I don't think these are meant to be correlated with individual frames, but I believe the system-time should line up with the sync-time on frames.

Reply:
That's a good design.  I would propose that if there's a need for frame level stats, they would be a different message that is tied to frame IDs and not system time.